test/tests/functional/pbs_reservations.py

```diff
@@ -1938,6 +1953,7 @@ def test_delete_idle_resv_job_standing(self):
                     'reserve_start': start_str}
         self.server.expect(RESV, exp_attr, id=rid, offset=idle_timer + 5)
 
+    @skipOnCpuSet
```

Comment:
As I understand the purpose of job j1 is to keep the complex busy so that j2 and j3 do not run.
We can set scheduling=False in the beginning of the test case and then submit two jobs.
Convert them to asap reservations. Would that work?

Reply:
@prakashcv13 , I tried your suggestion, but it wont work.

Reply:
I wouldn't expect this to work because the scheduler needs to run to confirm the ASAP reservations.  If scheduling is off, then it doesn't run.

Reply:
and that is why it didn't work :)